test/javascripts/components/load-more-test.js.es6

```diff
@@ -0,0 +1,28 @@
+import { _setTestParams } from "discourse/lib/eyeline";
+import componentTest from "helpers/component-test";
+
+moduleForComponent("load-more", { integration: true });
+
+componentTest("preview updates with markdown", {
+  template: `
+    {{#load-more selector=".numbers tr" action=(action loadMore)}}
+      <table class='numbers'>
+        <tr><td>1</td></tr>
+        <tr><td>2</td></tr>
+        <tr><td>3</td></tr>
+      </table>
+    {{/load-more}}`,
+
+  beforeEach() {
+    this.set("loadMore", () => this.set("loadedMore", true));
+    _setTestParams(false, true);
+  },
+
+  afterEach() {
+    _setTestParams(true);
+  },
+
+  async test(assert) {
```

Comment:
is it `async` though? if we don't use `await` inside?

Reply:
Nope. :)